common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -311,6 +344,42 @@ def refresh_children(self, request, suffix, update_db=True):  # pylint: disable=
         lib_tools.update_children(self, user_id, update_db)
         return Response()
 
+    def _validate_library_version(self, validation, lib_tools, version, library_key):
+        """
+        Validates library version
+        """
+        latest_version = lib_tools.get_library_version(library_key)
+        if latest_version is not None:
+            if version is None or version != latest_version:
+                validation.set_summary(
+                    StudioValidationMessage(
+                        StudioValidationMessage.WARNING,
+                        _(u'This component is out of date. The library has new content.'),
+                        # TODO: change this to action_runtime_event='...' once the unit page supports that feature.
+                        # See https://openedx.atlassian.net/browse/TNL-993
+                        action_class='library-update-btn',
+                        # Translators: ↻ is an UTF icon symbol, no need translating it.
+                        action_label=_(u"↻ Update now.")
```

Comment:
This came up in another review too. Please do move ↻ out of the translatable string, using a placeholder.